dts/arm/st/g4/stm32g4.dtsi

```diff
@@ -27,6 +27,26 @@
 	};
 
 	soc {
+		adc0: adc@50000000 {
```

Comment:
Can you add a comment stating that support of both adc instances running in parallel is not supported right now?

Reply:
Done.